clients/filesystem-hadoop3/src/main/java/com/datastrato/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -378,7 +378,9 @@ private Pair<Fileset, FileSystem> constructNewFilesetPair(NameIdentifier identif
 
   private Fileset loadFileset(NameIdentifier identifier) {
```

Comment:
The `NameIdentifier` in gvfs is still a full qualified name identifier, should you modify the logic also, not just change the client method here?

Reply:
In my understanding, the GVFS is a FS implementation, not a public API for end user; The methods like "loadFileset" are internal implementations, which are invisible to end users; For end users, they just use "gvfs://filesets/<catalog>/<schema>/<fileset>/" to access, which is always a full path; we're not going to provide a relative path or access method like gvfs://<schema>/<fileset>/, am I correct? 

Just let me know your thoughts here. Thank you!